libmozdata/patchanalysis.py

```diff
@@ -563,6 +563,21 @@ def attachmenthandler(attachments, bugid):
         if info['uplift_comment'] and info['uplift_author']:
             break
 
+    # Landing dates per useful channels
+    all_channels = ['nightly', 'aurora', 'beta', 'release']
```

Comment:
I'm not sure, I think in theory it could be either before or after. Perhaps we should just remove the `channels = all_channels[:all_channels.index(channel) + 1]` line and always find the landing dates for all channels.
In ship-it you can then just show the ones that are not None.